qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/QueryBuilderBWCIT.java

```diff
@@ -63,26 +63,26 @@
 
     static {
         addCandidate("""
-            "match": { "keyword_field": "value"}
```

Comment:
For reviewers: this test writes percolator queries into an old index, then upgrades and tests that reading the index gets the same queries out.  This commit changes how keyword fields are rewritten, so breaks the assumption in this test that the queries will look the same.  Changing the query so that it targets a text field instead of a keyword field preserves the constraint under test.